src/main/java/wolox/training/models/Book.java

```diff
@@ -1,21 +1,51 @@
 package wolox.training.models;
 
+import static com.google.common.base.Preconditions.checkArgument;
+import static wolox.training.utils.Utils.checkString;
+
+import io.swagger.annotations.ApiModel;
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import lombok.AccessLevel;
 import lombok.Data;
+import lombok.EqualsAndHashCode;
 import lombok.Setter;
 
 /**
  * Book is the main entity we'll be using for the java training
  *
  * @author tinoq-woloxer
  */
-@Entity @Data
+@Entity
+@Data
+@ApiModel(description = "Model of a book that may be assigned to users")
+@EqualsAndHashCode
 public class Book {
+    private Book(){}
```

Comment:
In Java classes we used to have first the class members, then the constructors and then the methods (Getters and setters includes).
[This link](https://docs.google.com/document/d/1xBoy41sG9vkrVsOd109OQ61LBQxPzZ2_F3K9e5jfHQY/edit?ts=5e4db74c#heading=h.3lunxfjzfi7d) could be useful :)